### PR TITLE
Switch from manage.py to flask cli

### PR DESCRIPTION
**Pull request by iambibhas**
Related to https://github.com/hasgeek/funnel/pull/1148

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ rm build/*.* && yarn build
 ```
 createdb boxoffice
 pip install -r requirements.txt
-./manage.py dbconfig | sudo -c postgres psql boxoffice
-./manage.py createdb
+flask dbconfig | sudo -c postgres psql boxoffice
+flask createdb
```
Comment (iambibhas) [inline README.md:26]: This does not exist right now. Is it coming from coaster?

Comment (jace) [reply README.md:26]: It's not required for the tests. They do createdb in setup.

File: boxoffice/cli.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from boxoffice import app
```
Comment (jace) [inline boxoffice/cli.py:1]: `from . import app`

**Review by jace — CHANGES_REQUESTED** [review]

File: setup.cfg [hunk 3]
```diff
@@ -10,7 +10,6 @@ omit =
     runserver.py
     runtestserver.py
     website.py
```
Comment (jace) [inline setup.cfg:12]: Delete website.py

**Review by jace — CHANGES_REQUESTED** [review]

**Pull request merged**